src/pat/contentbrowser/src/ContentBrowser.svelte

```diff
@@ -95,11 +95,20 @@
         });
     }
 
+    function isBrowseable(item) {
+        return (
+            item.is_folderish &&
+            (($config.browseableTypes.length &&
+                $config.browseableTypes.indexOf(item.portal_type) != -1) ||
+                !$config.browseableTypes.length)
```

Comment:
I did not try it, but this is shorter and should have the same effect:

```suggestion
            (!config.browseableTypes.length ||
                $config.browseableTypes.indexOf(item.portal_type) != -1)
```

Reply:
changed that ... thanks for the suggestion!